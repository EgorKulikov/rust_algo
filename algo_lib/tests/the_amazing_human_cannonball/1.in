11
19 45 20 9 12
20 45 20 9 12
25 45 20 9 12
20 43 20 9 12
20 47.5 20 9 12
20 45 17 9 12
20 45 24 9 12
20 45 20 10 12
20 45 20 9 11
20 45 20 9.0 11.5
20 45 18.1 9 12
