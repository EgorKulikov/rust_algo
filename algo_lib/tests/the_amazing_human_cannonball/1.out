Not Safe
Safe
Not Safe
Not Safe
Not Safe
Not Safe
Not Safe
Not Safe
Not Safe
Safe
Safe
